README.md

```diff
@@ -1 +1,17 @@
-# hw01
\ No newline at end of file
+# hw01
+
+My complete name is *Miguel Ángel Rosales Legorreta*
+[My linkedin profile](www.linkedin.com)
+
+I'm doing my Master's degree in Public Policy at Harris
+
+Before comming to Chicago I worked in Mexico for
+* __The  Ministry of Finance__
+    + at the International Affairs Unit
+    + at the Undersecretary office
+* __The Ministry of Social Development__
+    + at the Evalution Department
+* __The Minstry of Foreign Affaris__
+    + at the Mexican Agency of International Development Cooperation
+
+![Here is a picture of me!](https://m.media-amazon.com/images/M/MV5BMTkxMzk4MjQ4MF5BMl5BanBnXkFtZTcwMzExODQxOA@@._V1_UX214_CR0,0,214,317_AL_.jpg)
```

Comment:
Unfortunately, GitHub is not able to render captions, so we were not able to this 😞 . Put good attempt and great work embedding the image!